src/app/client/src/assets/libs/mathEquation/plugin/mathTextPlugin.js

```diff
@@ -0,0 +1,126 @@
+import { modelToViewAttributeConverter } from '@ckeditor/ckeditor5-image/src/image/converters';
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import ClickObserver from '@ckeditor/ckeditor5-engine/src/view/observer/clickobserver';
+var iframeObj;
+export default class MathText extends Plugin {
+
+    init() {
+        const editor = this.editor;
+        const view = editor.editing.view;
+        const componentFactory  = editor.ui.componentFactory;
+        this._defineSchema();
+        this._defineConverters();
+        view.addObserver( ClickObserver );
+
+        componentFactory.add('MathText', locale => {
+            const view = new ButtonView(locale);
+            view.set({
+                label: 'Add Equations',
+                withText: true,
+                tooltip: true
+            });
+
+            // Callback executed once the image is clicked.
+            this.listenTo(view, 'execute', () => {
+                this._loadIframeModal('btnClick');
+            });
+            return view;
+        });
+
+        this.listenTo( editor.editing.view.document, 'click', ( evt, data ) => {
+            if ( data.domEvent.detail == 3 ) {
+                let ifrm = document.getElementById('mathModalIframe')
+                if (data.domTarget.nodeName.toLowerCase() == 'img' && data.domTarget.hasAttribute("data-mathtext") && !ifrm) {
```

Comment:
@vinukumar-vs data-mathtext attribute is added to store the formula. This will help for the migration of existing questions. 
cc: @pallakartheekreddy @Nik720 @vaibhavbhuva